api/desecapi/pdns_change_tracker.py

```diff
@@ -93,7 +93,16 @@ def pdns_do(self):
                     'kind': 'MASTER',
                     'dnssec': True,
                     'nsec3param': '1 0 127 %s' % salt,
-                    'nameservers': settings.DEFAULT_NS
+                    'nameservers': settings.DEFAULT_NS,
+                    'rrsets': [{
+                        'name': self.domain_name_normalized,
+                        'type': 'SOA',
+                        'ttl': 300,
+                        'records': [{
+                            'content': 'set.an.example. get.desec.io. 1 1209600 1209600 2419200 3600',
```

Comment:
Let's document the rationale for each choice here or elsewhere (but in the code) https://github.com/desec-io/desec-stack/issues/337#issuecomment-623488329

Reply:
done